app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -74,9 +74,17 @@ function DocumentCaptureReviewIssues({
           ]}
         />
       )}
```

Comment:
@kellular - @eileen-nava wisely recommended I tag you in this as well as it is going to affect the UX for selfie capture experience (it should not affect the non-selfie experience).

The issue:

[This ticket](https://cm-jira.usa.gov/browse/LG-12307) is to prevent uploads from being possible if selfie capture is enabled. It also has an AC of preventing use of the native camera.

As it is, this will create what I'd likely call a bad user experience for anyone struggling to capture a good image. This is what happens:

- A user tries taking images with the SDK
- They fail 3 times
- They are stuck on a screen with no error message and no reason they can see why they are stuck
- They can still click on the image field or the "Take photo" button, but nothing happens (I actually see a little jitter the fourth time I click "Take photo" after a failure, and then all subsequent tries have nothing happen)

Possible solutions that I see:

- A comment on the ticket says "Will defer question about SDK capture limits to a separate ticket." Perhaps since this is only affecting the selfie flow, it's okay if it's a little broken until that ticket is resolved? But it's unknown how long that would take, so if we go that route it could remain broken in staging for a while.
- Try to display some sort of error message. I'm unsure how easy or straightforward this will be.
- Temporarily remove SDK capture limits if selfie capture is enabled. Unsure exactly how hard / easy this will be but it may be easier than an error message

Perhaps you might have other ideas I haven't thought of as well. Curious to hear your thoughts!

Reply:
Hey Brittany, thanks for the tag and flagging these! I caught up with @daviddsilvanava and it's ok to merge your Pull Request. We have planned tickets in the backlog that address the parts of the user experience you are pointing out here:
1. [Remove the SDK capture limits](https://cm-jira.usa.gov/browse/LG-12511)
2. [Figure out the UX for when a user cannot capture images with the SDK in the biometric flow](https://cm-jira.usa.gov/browse/LG-12433)

Reply:
Thank you so much, @kellular!

This probably won't impact you too much, but just FYI: I'm trying to tackle this work in a different way. If the other way works, I will likely not merge this PR, but that one instead. It should have the same result / UX issues though - I will tag you and link this comment chain on the new PR once it's up, just for your awareness.

Reply:
Ok sounds good, I appreciate you letting me know!